web/public/locales/de.json

```diff
@@ -98,6 +98,40 @@
       "pill": "Nicht verfügbar",
       "body": "Unsere Datenquellen stellen aktuell keine Werte für diese Zone zur Verfügung. Die angezeigten Werte sind Schätzungen, die auf historischen Messungen basieren und durch verifizierte Daten ersetzt werden, sobald diese verfügbar sind. Bitte beachten Sie, dass geschätzte Werte nicht immer genau sind, da sie aktuelle Wetterereignisse nicht berücksichtigen."
     },
+    "estimated_mode_breakdown": {
+      "title": "Teils geschätzte Daten",
+      "pill": "Unvollständig",
+      "body": "Die Daten zur Stromerzeugung in diesem Gebiet sind unvollständig. Die fehlenden Werte wurden anhand eines statistischen Schätzmodells berechnet."
+    },
+    "estimated_reconstruct_breakdown": {
+      "title": "Ausschließlich geschätzte Daten",
+      "pill": "Unpräzise",
+      "body": "Die veröffentlichten Daten für diese Zone geben nur die Gesamtproduktion an. Die erneuerbaren Quellen werden anhand aktueller Wetterdaten geschätzt, während die fossilen Quellen auf der Grundlage historischer Daten für jede Produktionsquelle abgezogen werden."
+    },
+    "estimated_generic_method": {
+      "title": "Daten werden geschätzt",
+      "pill": "Unpräzise",
+      "body": "Die veröffentlichten Daten für diese Zone sind nicht verfügbar oder unvollständig. Die auf der Karte gezeigten Daten sind nach bestem Wissen und Gewissen geschätzt, können aber von den tatsächlichen Werten abweichen."
+    },
+    "estimated_construct_breakdown": {
+      "title": "Ausschließlich geschätzte Daten",
+      "pill": "Nicht in Echtzeit",
+      "body": "Die Daten für diese Zone werden nicht in Echtzeit veröffentlicht und geben nur die Gesamtproduktion an. Sowohl die stündliche Produktion als auch die Produktionsquellen werden auf der Grundlage historischer Daten für jede Produktionsquelle geschätzt."
+    },
+    "estimated_construct_zero_breakdown": {
+      "title": "Ausschließlich geschätzte Daten",
+      "pill": "Nicht in Echtzeit",
+      "body": "Die Daten für diese Zone werden nicht in Echtzeit veröffentlicht und geben nur die Gesamtproduktion an. Sowohl die stündliche Produktion als auch die Produktionsquellen werden auf der Grundlage historischer Daten für jede Produktionsquelle geschätzt."
+    },
+    "threshold_filtered": {
+      "title": "Fehlende Daten",
+      "pill": "Ausfall von Daten",
```

Comment:
```suggestion
      "pill": "Ausfall der Datenübermittlung",
```

Reply:
I think this pill text would be too long so I kept the old text